cmp/grid/Grid.js

```diff
@@ -177,6 +178,10 @@ class GridLocalModel extends HoistModel {
         this.addReaction(this.sizingModeReaction());
         this.addReaction(this.validationDisplayReaction());
 
+        // If ancestor of a modalSupport cmp:
```

Comment:
no need for comment here,  the method itself is perfectly clear, and the benefits of having the method mean we don't need to distract in the constructor.